docs/content/Caching/Overview.md

```diff
@@ -126,7 +126,7 @@ cube(`Orders`, {
 
 By default, Cube.js will check and invalidate the cache in the background when
 in [development mode][link-development-mode]. When development mode is disabled
```

Comment:
In the production, we recommend running Refresh Worker as a separate instance _and link to the refresh worker section in deployment overview._